CODEOWNERS

```diff
@@ -224,6 +224,7 @@
 /CODE_OF_CONDUCT.md @cilium/contributing
 /CODEOWNERS @cilium/contributing
 /CONTRIBUTING.md @cilium/contributing
+/CHANGELOG.md @cilium/release-managers
```

Comment:
This one actually only exists in stable release branches, but it comes up when preparing the prereleases... The GitHub action for validating CODEOWNERS fails on this line though because it appears to be stale. Probably need to figure out how to create an exception in the GHA if we want this to generally hold true.